it.sh

```diff
@@ -18,17 +18,20 @@
 
 # Utility script for running the new integration tests, since the Maven
 # commands are unwieldy.
+set -e
 
 export DRUID_DEV=$(cd $(dirname $0) && pwd)
 
 function usage
 {
-	cat <<EOF
+  cat <<EOF
```

Comment:
What is our standard indentation in scripts? It is 2 in Java. I think I saw it as 4 in other scripts, though I might be mistaken.

Reply:
There was a mix of tabs and spaces in the script. I just replaced all the tabs with spaces to keep it consist. I think, IntelliJ decided it should to 2 spaces for shell script.